pkg/fake/pricingapi.go

```diff
@@ -51,6 +51,10 @@ func (p *PricingAPI) GetProductsPagesWithContext(_ aws.Context, _ *pricing.GetPr
 }
 
 func NewOnDemandPrice(instanceType string, price float64) aws.JSONValue {
+	return NewOnDemandPriceInCurrency(instanceType, price, "USD")
+}
+
+func NewOnDemandPriceInCurrency(instanceType string, price float64, currency string) aws.JSONValue {
```

Comment:
Just one nit, outside of that looks good to me!

```suggestion
func NewOnDemandPriceWithCurrency(instanceType string, price float64, currency string) aws.JSONValue {
```

Reply:
Done. Thanks for your review.